include/cantera/oneD/Domain1D.h

```diff
@@ -323,6 +324,12 @@ class Domain1D
      */
     virtual void restore(const AnyMap& state, double* soln, int loglevel);
 
+    //! Return reference to thermo/kinetics/transport manager
+    //! @since  New in Cantera 3.0.
+    shared_ptr<Solution> sol() const {
+        return m_sol;
+    }
```

Comment:
Since this is new, can we use the full `m_solution` and `solution()`? I think it'll probably be inconsistent with other interfaces but we're not working with 80-character punch cards anymore...

Reply:
no objections to `m_solution`, but there's an unfortunate clash for `solution()`: https://github.com/Cantera/cantera/blob/177356c6687f135f2b95cf566d803f948a032a10/include/cantera/oneD/Sim1D.h#L109-L111

... it's a slightly different context, but I'd like to avoid a clash of nomenclatures

Reply:
That's disappointing. Even with the 3.0 break, I still think changing the functionality of the existing `solution()` so drastically as to return a `Solution` would need to go through a deprecation cycle, plus whatever we use for the `m_solution`-getter would immediately be deprecated... I don't have any other good suggestions 😦 

As another nitpick on these lines, should the comment say `Return a _pointer_ to thermo/kinetics/transport manager`?

Reply:
Maybe `getSolution()`? It's still a little confusing, but I'm not sure that can be entirely avoided.

Regarding the comment, I'd just say "Get the thermo/kinetics/transport managers used in the domain" -- I don't think it's necessary or helpful to repeat the information about the type that's more precisely given in the function signature.

Reply:
As `Sim1D::solution` is unused at this point, I decided to deprecate it (together with `Sim1D::setSolution` and use `Domain1D::solution` as it is consistent with the nomenclature we use for getters.